src/components/task-list/index.md.njk

```diff
@@ -0,0 +1,145 @@
+---
+title: Task list
+description: The task list component displays all the tasks a user needs to do, and allows users to easily identify which ones are done and which they still need to do.
+section: Components
+aliases:
+backlog_issue_id: 72
+layout: layout-pane.njk
+---
+
+{% from "_example.njk" import example %}
+
+The task list component displays all the tasks a user needs to do, and allows users to easily identify which ones are done and which they still need to do.
+
+{{ example({group: "components", item: "task-list", example: "default", id: "default-1", html: true, nunjucks: true, open: false}) }}
```

Comment:
Nice extra for this one

We use iframe `loading="lazy"` by default but set the first example to `loading="eager"`

```suggestion
{{ example({ group: "components", item: "task-list", example: "default", id: "default-1", html: true, nunjucks: true, open: false, loading: "eager" }) }}
```